src/test/java/de/tum/in/www1/artemis/lecture/AttachmentResourceIntegrationTest.java

```diff
@@ -149,4 +163,29 @@ void deleteAttachment_notInstructorInACourse() throws Exception {
         lectureRepository.save(lecture);
         request.delete("/api/attachments/" + attachment.getId(), HttpStatus.FORBIDDEN);
     }
+
+    private MockHttpServletRequestBuilder buildCreateAttachment(@NotNull Attachment attachment) throws JsonProcessingException {
+        return buildCreateAttachment(attachment, null);
+    }
+
+    private MockHttpServletRequestBuilder buildCreateAttachment(@NotNull Attachment attachment, String fileContent) throws JsonProcessingException {
+        var attachmentPart = new MockMultipartFile("attachment", "", MediaType.APPLICATION_JSON_VALUE, objectMapper.writeValueAsBytes(attachment));
```

Comment:
Can we extract some part of the method to the RequestUtilService, so we don't always have to reimplement the multipart thing when using this in other endpoints as well